scripts/dotnet-cli-build/Utils/DotNetCli.cs

```diff
@@ -42,5 +51,19 @@ private static string GetStage0Path()
                 return Path.Combine(Directory.GetCurrentDirectory(), ".dotnet_stage0", PlatformServices.Default.Runtime.OperatingSystemPlatform.ToString(), "share", "dotnet", "cli", "bin");
             }
         }
+
+        // Mega-hack because we can't take a reference on a project within the build script
+        internal static IEnumerable<string> GetOverrideRestoreRuntimeIdentifiers(IRuntimeEnvironment env)
```

Comment:
This will be removed in #1639 very shortly after merging this PR (once it hits stage 0)